composer.json

```diff
@@ -5,7 +5,7 @@
     "keywords": ["framework", "laravel"],
     "license": "MIT",
     "require": {
-        "php": "^8.0.2",
+        "php": "^8.1",
```

Comment:
Brave to change during developing the php version. Did you test all of the backend to ensure, that everything is still working?

Reply:
changed back, was changed by accident